src/styles/style.css

```diff
@@ -0,0 +1,7 @@
+/* add styles here */
+
+@import 'blocks/stars.css';
+
+.page {
+  margin: 0;
```

Comment:
Be consistent with your margins. It's recommended to add margins on the top or bottom only, but not both, to avoid potential margin collapse issues.